art/__init__.py

```diff
@@ -4,11 +4,11 @@
 import logging.config
 
 # Project Imports
-from art import attacks
+# from art import attacks
 from art import defences
-from art import estimators
-from art import metrics
```

Comment:
After removing, the following won't work:
```
import art
art.metrics
```
and will result in an `AttributeError: module 'art' has no attribute 'metrics'`.

Is this intended?

Reply:
No, this was not intended and needs to be fixed.